lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
Right. Nothing [here](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/report/templates/report-template.html#L228-L247) ends up in Viewer. Because of that, we should avoid adding things to `getReport*` that are specific to the main report and never used elsewhere. This is a good example, where the init code is specific to the page. Otherwise the browserified bundle gets larger :\